src/js/components/DataTable/Cell.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { TableCell } from '../Table';
+import { Box } from '../Box';
+import { Text } from '../Text';
+
+const Cell = ({
+  column: { align, property, primary, render }, context, datum, scope, theme,
+  ...rest
+}) => {
+  let content;
+  if (render) {
+    if (datum[property]) {
+      content = render(datum);
+    }
+  } else if (datum[property]) {
+    content = datum[property];
+  }
+
+  if (typeof content === 'string' || typeof content === 'number') {
+    if (primary) {
+      content = <strong>{content}</strong>;
+    }
+    content = <Text>{content}</Text>;
+  }
+
+  if (theme.dataTable[context]) {
+    content = (
+      <Box
+        direction='row'
+        justify={align}
+        style={{ height: '100%' }}
```

Comment:
should we use Box height prop ?

Reply:
Good suggestions. Box doesn't support this today, but I think it should.